internal/common.go

```diff
@@ -47,3 +52,25 @@ const (
 	// BundleRootfsName is the name of the bundle's directory for holding the container's rootfs
 	BundleRootfsName = "rootfs"
 )
+
+// MagicStubBytes used to determine whether or not a drive is a stub drive
+var MagicStubBytes = []byte{214, 244, 216, 245, 215, 177, 177, 177}
+
+// IsStubDrive will check to see if the io.Reader follows our stub drive
+// format. In the event of an error, this will return false. This will only
+// return true when the first n bytes read matches that of the magic stub
+// bytes and where n is the length of magic bytes.
+func IsStubDrive(r io.Reader) bool {
+	buf := make([]byte, len(MagicStubBytes))
+	lr := io.LimitReader(r, 512)
```

Comment:
Why are you setting the limit to 512 and not len(MagicStubBytes)?

Reply:
Changed it to the `len(MagicStubBytes)`. Think this was a brain fart.